src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.DefaultOutputPaths.targets

```diff
@@ -32,14 +32,14 @@ Copyright (c) .NET Foundation. All rights reserved.
     
     <BaseOutputPath Condition="'$(BaseOutputPath)' == ''">bin\</BaseOutputPath>
     <BaseOutputPath Condition="!HasTrailingSlash('$(BaseOutputPath)')">$(BaseOutputPath)\</BaseOutputPath>
-    <OutputPath Condition="'$(OutputPath)' == '' and '$(PlatformName)' == 'AnyCPU'">$(BaseOutputPath)$(Configuration)\</OutputPath>
-    <OutputPath Condition="'$(OutputPath)' == '' and '$(PlatformName)' != 'AnyCPU'">$(BaseOutputPath)$(PlatformName)\$(Configuration)\</OutputPath>
+    <OutputPath Condition="'$(OutputPath)' == '' and '$(PlatformName)' == 'AnyCPU'">$(BaseOutputPath)$(Configuration).ToLowerInvariant()\</OutputPath>
```

Comment:
thank you been a while since I did any msbuild @dsplaisted for some reason in linux the build is not working right with a clean checkout it check out the gist for the build out put: https://gist.github.com/dmportella/38d7ded5cd46747afb2f047a944df510

I will push the new changes